src/sv-pipeline/scripts/aggregate_picard.py

```diff
@@ -0,0 +1,276 @@
+                                        'WORST_PRE_CXT_9', 'WORST_PRE_CXT_QSCORE_9', 'WORST_POST_CXT_9',
+                                       'WORST_POST_CXT_QSCORE_9'], axis = 1)
+    sadften = pd.DataFrame(table.loc[10]).T.reset_index(drop = True)
+    sadf10 = sadften.add_suffix("_10").drop(['SAMPLE_ALIAS_10', 'LIBRARY_10', 
+                                         'WORST_CXT_10', 'WORST_CXT_QSCORE_10', 
+                                         'WORST_PRE_CXT_10', 'WORST_PRE_CXT_QSCORE_10', 'WORST_POST_CXT_10',
+                                       'WORST_POST_CXT_QSCORE_10'], axis = 1)
+    sadfeleven = pd.DataFrame(table.loc[11]).T.reset_index(drop = True)
+    sadf11 = sadfeleven.add_suffix("_11").drop(['SAMPLE_ALIAS_11', 'LIBRARY_11', 
+                                            'WORST_CXT_11', 'WORST_CXT_QSCORE_11', 
+                                            'WORST_PRE_CXT_11', 'WORST_PRE_CXT_QSCORE_11', 'WORST_POST_CXT_11',
+                                       'WORST_POST_CXT_QSCORE_11'], axis = 1)
+    sadftwelve = pd.DataFrame(table.loc[12]).T.reset_index(drop = True)
+    sadf12 = sadftwelve.add_suffix("_12").drop(['SAMPLE_ALIAS_12', 'LIBRARY_12', 
+                                            'WORST_CXT_12', 'WORST_CXT_QSCORE_12', 'WORST_PRE_CXT_12', 
+                                            'WORST_PRE_CXT_QSCORE_12', 'WORST_POST_CXT_12',
+                                       'WORST_POST_CXT_QSCORE_12'], axis = 1)
+    SAMresult = pd.concat([sadf1,sadf2, sadf3, sadf4, sadf5, sadf6, sadf7, sadf8, sadf9, sadf10, sadf11, sadf12 ], axis = 1)
+    return SAMresult
+
+def windows(table, window):
+    if len(table.columns) == 2:
+        return window_metrics_mean(table, window)
+    
+    if len(table.columns) == 7:
+        return window_metrics_std(table, window)
+        
+
+def mainfile(index):
+    allsamplecolumns = ['alignment_summary_metrics', 'base_distribution_by_cycle_table', 'gc_bias_summary_metrics', 
+                        'insert_size_metrics', 'mean_quality_by_cycle_table', 
+                        'sequencing_artifact_summary_metrics', 'quality_score_table', 'quality_yield_metrics', 
+                        'raw_wgs_metrics', 'wgs_metrics']
+    table_name = "sample"
+    samples = pd.read_csv(io.StringIO(fiss.fapi.get_entities_tsv(project, workspace, 'sample').text), sep='\t')
+    samples.rename(columns = {'entity:sample_id':'sample'}, inplace = True)
+#     specificcolumns = samples[['alignment_summary_metrics', 'base_distribution_by_cycle_table', 'gc_bias_summary_metrics', 
+#                         'insert_size_metrics', 'mean_quality_by_cycle_table', 
+#                         'sequencing_artifact_summary_metrics', 'quality_score_table', 'quality_yield_metrics', 
+#                         'raw_wgs_metrics', 'wgs_metrics']]
+#     dropemptyrows = specificcolumns.dropna(axis = 0)
+#     dropemptycolumns = dropemptyrows.dropna(axis = 1)
+#     files = ! ls
+    
+    Dict = {}
```

Comment:
Make the variable name more descriptive and lower case. `dict` is also name for a built-in type in Python and typing library has a class `Dict` so it's best to avoid using either.